KsApi/models/graphql/GraphBackingTests.swift

```diff
@@ -22,7 +22,18 @@ final class GraphBackingTests: XCTestCase {
               "id": "QmFja2luZy0xMTMzMTC5ODE=",
               "uid": "565656",
               "name": "Backer McGee",
-              "imageUrl": "http://www.kickstarter.com/avatar.jpg"
+              "imageUrl": "http://www.kickstarter.com/avatar.jpg",
```

Comment:
Because the `UserFragment` was updated and it's referenced in other queries/fragment, other than the `FetchUser` query, lots of tests and stub data needed to be updated. This was a lot of tedious work.

Reply:
glad you got it working!

Reply:
what ended up being the issue with the `missingValue`?

Reply:
turns out by keeping `CreditCardConnection` which had a property called storedCards too the decoder was expecting two `storedCards` keys instead of the one actually returning from the response. took me far longer than im willing to admit to catch that.